src/styles/main.scss

```diff
@@ -0,0 +1,1590 @@
+@import "vendor-port.css";
```

Comment:
How is this file different from `base.scss`? Doesn't `base.scss` serve a similar purpose?
@Sanji515 may also help in the discussion here.